src/app/services/tasks-by-status.service.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { TaskStatus } from '@aneoconsultingfr/armonik.api.angular';
+import { TestBed } from '@angular/core/testing';
+import { TaskStatusColored } from '@app/types/dialog';
+import { DefaultConfigService } from './default-config.service';
+import { StorageService } from './storage.service';
+import { TasksByStatusService } from './tasks-by-status.service';
+
+
+
+describe('tasksByStatusService', () => {
+  let service: TasksByStatusService;
+  const mockStorage = {
+    getItem: jest.fn(),
+    setItem: jest.fn(), 
+    'applications-tasks-by-status': [{'color': '#4caf50', 'status': 4}],
+  };
+  const mockTaskStatusColored: TaskStatusColored[] = [
+    {
+      status: TaskStatus.TASK_STATUS_COMPLETED,
+      color: '#4caf50',
+    },
+    {
+      status: TaskStatus.TASK_STATUS_ERROR,
+      color: '#ff0000',
+    },
+    {
+      status: TaskStatus.TASK_STATUS_TIMEOUT,
+      color: '#ff6944',
+    },
+    {
+      status: TaskStatus.TASK_STATUS_RETRIED,
+      color: '#ff9800',
+    },
+  ];
+
+  beforeEach(() => {
+    service = TestBed.configureTestingModule({
+      providers: [
+        TasksByStatusService,
+        DefaultConfigService,
+        {provide: StorageService, useValue: mockStorage}
+         
+      ]
+    }).inject(TasksByStatusService);
+  });
+
+  test('it should create tasks by status', () =>{
+    expect(service).toBeTruthy();
```

Comment:
```suggestion
  it('should create tasks by status', () =>{
    expect(service).toBeTruthy();
```

Apparently there is no difference between `it`and `test`. Our choice to use one or the other depends on the readability of our tests (ref [stackOverflow](https://stackoverflow.com/questions/45778192/what-is-the-difference-between-it-and-test-in-jest#47399770)).
When our tests start with the word "should", it might be better to use `it`.